docs/.vale/writing-styles/expo-docs/Consistency.yml

```diff
@@ -8,7 +8,7 @@ swap:
   amazon appstore: Amazon Appstore
   android emulator: Android Emulator
   apple developer: Apple Developer
-  cancelation: cancelation
+  cancellation: cancelation
```

Comment:
https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#spelling-and-word-choices

Reply:
Ah, my bad. This needs to be reversed. Thanks for catching this.